pkg/kubelet/kubelet.go

```diff
@@ -1680,7 +1674,6 @@ func (kl *Kubelet) deletePod(pod *v1.Pod) error {
 	podPair := kubecontainer.PodPair{APIPod: pod, RunningPod: &runningPod}
 
 	kl.podKiller.KillPod(&podPair)
-	// TODO: delete the mirror pod here?
```

Comment:
we cleaned up mirror pod deletion flow (including grace period) via: https://github.com/kubernetes/kubernetes/pull/92442
https://github.com/kubernetes/kubernetes/pull/93307